packages/modules/src/scoring/Scoring.ts

```diff
@@ -0,0 +1,68 @@
+import { SCORE_MODULUS, IJournalContent,IRatingInput,CordKeyringPair } from '@cord.network/types'
+import { Crypto } from '@cord.network/utils'
+import {
+  SCORE_IDENT,
+  SCORE_PREFIX,
+  ScoreType,
+  IScoreAverageDetails,
+} from '@cord.network/types'
+import { Identifier } from '@cord.network/utils'
+import { fetchScore } from './Scoring.chain'
+
+export function adjustAndRoundRating(rating: number): number {
+  rating = Math.round(rating * SCORE_MODULUS)
+  return rating
+}
+
+export function ComputeActualRating(rating: number): number {
+  return rating / SCORE_MODULUS
+}
+
+export function ComputeAverageRating(rating: number, count: number): number {
+  return rating / count
+}
+
+export function generateDigestFromJournalContent(
+  journalContent: IJournalContent
+) {
+  const digest = Crypto.hash(JSON.stringify(journalContent))
```

Comment:
Look at the Schema and Registry implementation of creating Hash and Identifier. We should follow the same methods.